documentation/Corpus-documentation.md

```diff
@@ -0,0 +1,19 @@
+# Corpus documentation pages
+
+Corpora can include markdown templates containing documentation. I-analyzer supports several types of documentation pages.
+
+- general information
+- citation guidelines
+- documentation on word models
+- a licence for re-using the data
+- terms of service (usually pro)
```

Comment:
Define `usually pro` ?

Reply:
ah, I think I just walked away from the computer halfway through that sentence.... 😅